scripts/core/world/world.js

```diff
@@ -136,6 +141,14 @@ define([
                     : {};
             }
 
+            this.setAgentCallback = function(cb) {
```

Comment:
That's the method you have to call. 
And pass on the callback.
The method will be of type: 
`function(agent, newLocation, oldLocation)`

@jacklehamster